src/Processor/Gotenberg.php

```diff
@@ -159,6 +159,14 @@ public function getPdfFromString(string $html, array $params = [], bool $returnF
             $chromium->extraHttpHeaders($params['extraHttpHeaders']);
         }
 
+        if (isset($params['metadata'])) {
+            if (method_exists($chromium, 'metadata')) {
+                $chromium->metadata($params['metadata']);
+            } else {
+                $chromium->formValue('metadata', json_encode($params['metadata']));
+            }
```

Comment:
```suggestion
            if (method_exists($chromium, 'metadata')) {
                $chromium->formValue('metadata', json_encode($params['metadata']));
            }
```
Thank you both for suggesting the changes.

It seems that this way might be slightly better, "less prettier" than using the direct method, but at least is avoiding `method_exists`.

Reply:
I thought of that but wanted to avoid the `TODO` :) 
Looks much cleaner I guess! Thanks